### DISCUSSION TITLE
RLC-820A not streaming

### DISCUSSION THREAD

**Discussion by roberto-formulatrix** [koush/scrypted#1411] [General]
I've setup my Reolink RLC-820A wit Scrypted Reolink/Plugin and I cannot get it to stream  I only see one frame and it never updates (until I refresh the page).  To test RSTP I tried VLC and worked fine without a problem,  of course the reolink app and the web client work fine.
RTMP/RSTP/ONVIF all on with the camera, https is off, and no funny chars in the password.

The last log in says:
[Rebroadcast Plugin]: parsed bitstream sps { width: 2560, height: 1440 }

The funny thing is that the camera is sending motionDetected events!

Ideas?

**Comment by rdrcrmatt** [comment]
Having the same issue.  According to streaming the camera in VLC the 4k stream is H.265.    Under Camera/Stream/Clear I changed the resolution to 2560*1440 and got H264.  The camera was immediately streamable in the Scrypted WebUI and in HomeKit on my phone.

**Reply by roberto-formulatrix** [reply]
Thank you.  Tried that, no luck... I'm streaming at 1440p already.  One extra thing that I did -- in come wireshark, and noticed all these RTMP packets that show unknown.  To be clear, just so you understand my setup:

scrypted vm (on proxmox) -> linksys switch -> unifi router ->linksys wireless -> ddwrt wireless bridge -> camera.  I'm guessing something is getting on the way there?  But I think I had it working before I added unifi, previously it was like this:
scrypted vm (on proxmox) -> linksys switch -> linksys wireless / router -> ddwrt wireless bridge -> camera
![image](https://github.com/koush/scrypted/assets/4512546/e9965d8f-82d1-4c6a-8a7d-047d2678dca3)

**Reply by roberto-formulatrix** [reply]
Strangely on exactly the same machine I can open rtmp streams using VLC

**Reply by rdrcrmatt** [reply]
What is the video format when you open the stream in VLC?
<img width="654" alt="image" src="https://github.com/koush/scrypted/assets/25355722/113ae753-c276-4181-9426-ded67f7e9eac">
